src/wrapped_radix_engine_toolkit/high_level/manifest_building/manifests_create_tokens.rs

```diff
@@ -434,21 +445,18 @@ CALL_METHOD
     }
 
     #[test]
-    fn create_single_nft_collection_assert_manifest() {
-        // This is fun! Below we use `#"` multi-line string literal in Rust,
-        // however, since string format of NonFungibleLocalId::Integer is "#,
-        // it makes a hassle, so I'm lazy and use `NonFungibleLocalId:Str` for test
-        // instead of integer to avoid this :D
+    fn create_two_nft_collections_assert_manifest() {
         let manifest =
-            TransactionManifest::create_non_fungible_tokens_collections_with_local_id_fn(
+            TransactionManifest::create_non_fungible_tokens_collections(
                 &AccountAddress::sample_stokenet(),
                 2,
                 2,
-                |i| NonFungibleLocalId::string(format!("{i}")).unwrap()
             );
-        let expected_manifest = r#"
+        let expected_manifest = r##"
```

Comment:
just learned about `r#####` allowing for us to use " and # inside the multiline string literal.